specification/compute/resource-manager/Microsoft.Compute/stable/2022-01-03/GalleryRP/communityGallery.json

```diff
@@ -325,6 +463,14 @@
           "type": "string",
           "format": "date-time",
           "description": "The end of life date of the gallery image version Definition. This property can be used for decommissioning purposes. This property is updatable."
+        },
+        "excludeFromLatest": {
```

Comment:
>excludeFromLatest [](http://example.com/codeflow?start=9&length=17)

ARM recommends string enums instead of booleans. Rather than adding booleans for selection filters, you could think of an include or exclude property with enum values, e.g. include: { All, AllButLatest }. If you use modelAsString: true, then over time you can add more enum values without adding a new API version.

Reply:
> 

these are the same api model in the backend as https://github.com/kangsun-ctrl/azure-rest-api-specs/blob/4d0adaa99964df094beae44cecb08f01ea6e66e0/specification/compute/resource-manager/Microsoft.Compute/stable/2021-10-01/gallery.json#L2440-L2443. So I think we can try to change them all in the new api version since we need this release go for our public preview.